modules/deployments-deploymentconfigs.adoc

```diff
@@ -30,7 +30,7 @@ The {product-title} `DeploymentConfig` object defines the following details:
 . Lifecycle hooks.
 
 Each time a deployment is triggered, whether manually or automatically, a deployer pod manages the deployment (including scaling down the old
-replication controller, scaling up the new one, and running hooks). The deployment pod remains for an indefinite amount of time after it completes the deployment in order to retain its logs of the deployment. When a deployment is superseded by another, the previous replication controller is retained to enable easy rollback if needed.
+replication controller, scaling up the new one, and running hooks). The deployment pod remains for an indefinite amount of time after it completes the deployment to retain its logs of the deployment. When a deployment is superseded by another, the previous replication controller is retained to enable easy rollback if needed.
```

Comment:
Another unsure if this meaning is the same

Reply:
I think this one makes sense as-is